atom/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -264,8 +264,8 @@ OffScreenRenderWidgetHostView::OffScreenRenderWidgetHostView(
 
   if (content::GpuDataManager::GetInstance()->HardwareAccelerationEnabled()) {
     video_consumer_.reset(new OffScreenVideoConsumer(
-        this, base::Bind(&OffScreenRenderWidgetHostView::OnPaint,
-                         weak_ptr_factory_.GetWeakPtr())));
+        this, base::BindRepeating(&OffScreenRenderWidgetHostView::OnPaint,
```

Comment:
/cc @zcbenz for osr review, is there any potential `OnceCallback` in this file ?

Reply:
It seems that all callbacks are paint and timer callbacks, and they should all be repeating.